packages/perseus-editor/src/components/interactive-graph-settings.tsx

```diff
@@ -80,11 +123,6 @@ class InteractiveGraphSettings extends React.Component<Props, State> {
     }
 
     static defaultProps = {
-        editableSettings: ["graph", "snap", "image", "measure"],
```

Comment:
Removing `editableSettings` because it's never actually passed down by InteractiveGarphEditor.

Reply:
Agreed! I think at this point anything that's unused we can classify as "didn't get implemented" and clean it up. Then when we start adding features we have a clean base to start from.